api/src/main/java/org/openmrs/util/OpenmrsConstants.java

```diff
@@ -1307,6 +1308,13 @@ public static String getOpenmrsProperty(String property) {
 	/** Value for the long person name format */
 	public static final String PERSON_NAME_FORMAT_LONG = "long";
 	
+	public static final String PLATFORM_UI_LOGIN_URL = "owa/openmrs-owa-platformui-0.1.0/index.html#/login";
+	
+	public void setDefaultLoginUrl_GP() {
+	Context.getAdministrationService().getGlobalPropertyObject(GP_LOGIN_URL).setPropertyValue(PLATFORM_UI_LOGIN_URL);
+	}
+	
```

Comment:
Thank you @mozzy11 for the work. Did you test this?

Reply:
@Ruhanga  , you mean testing it on my local installtion?? yes i did

Reply:
Can you also attach screen shots reflecting your changes as advised here https://wiki.openmrs.org/display/docs/Pull+Request+Tips ?

Reply:
hi @Ruhanga , i made a small update in the PR , thanks . i have attached the image showing the default Global property value changed to point to the platform-ui Login url

Reply:
updated commit is here  https://github.com/openmrs/openmrs-core/pull/2841/commits/5907c1b9806582a74dc88957ab6d95ce3ae11b95

Reply:
@mozzy11 , what happens when a User logs in with the set set Login url? How does the platform UI login page look like? What happens after a user logs in?